### ISSUE TITLE
Upgrade vm2 to fix critical severity vulnerability

### ISSUE THREAD

**Issue by SamDuvall** [jsreport/jsreport#873]
https://github.com/advisories/GHSA-6pw2-5hjv-9pf7

I tried upgrading `vm2` in my local package.json via "resolutions", but I end up with the following error.

```
TypeError: Error when evaluating engine handlebars for template anonymous
Cannot read properties of undefined (reading 'Contextify')
```

I am using the latest/following jsreport packages / versions

```
@jsreport/jsreport-chrome-pdf: 3.1.0
@jsreport/jsreport-core: 3.4.1
@jsreport/jsreport-handlebars: 3.0.0
@jsreport/jsreport-pdf-utils: 3.3.0
```

**Comment by pofider** [comment]
Thanks, we are checking on it.

**Comment by pofider** [comment]
We've shipped security hotfix jsreport@3.4.1 and @jsreport/jsreport-core@3.4.2 with the updated vm2.
